sdk/appconfiguration/app-configuration/src/internal/helpers.ts

```diff
@@ -459,3 +460,22 @@ export function hasUnderscoreResponse<T extends object>(
 ): result is T & HttpResponseField<any> {
   return Object.prototype.hasOwnProperty.call(result, "_response");
 }
+
+/**
+ * Get the scope for the App Configuration service based on the endpoint and audience.
+ * If the audience is provided, it will be used as the scope.
+ * If not, the scope is defaulted to Azure Public Cloud when not specified.
+ *
+ * @internal
+ */
+export function getScope(appConfigEndpoint: string, appConfigAudience?: string): string {
+  if (appConfigAudience) {
+    return `${appConfigAudience}/.default`;
+  } else if (appConfigEndpoint.includes("azure.us")) {
+    return `${KnownAppConfigAudience.AzureGovernment}/.default`;
+  } else if (appConfigEndpoint.includes("azure.cn")) {
+    return `${KnownAppConfigAudience.AzureChina}/.default`;
+  } else {
```

Comment:
I don't like behavior conditional on the endpoint URL, too fragile in the presence of proxies, DNS lookup, etc.